04_setup_ironic.sh

```diff
@@ -160,7 +160,7 @@ then
   sudo podman run -d --net host --privileged --name ipa-downloader --pod ironic-pod \
      -v $IRONIC_DATA_DIR:/shared ${IRONIC_IPA_DOWNLOADER_LOCAL_IMAGE} /usr/local/bin/get-resource.sh
 
-  sudo podman wait -i 1000ms ipa-downloader
+  sudo podman wait -i 1000 ipa-downloader
```

Comment:
Since it seems there's no format currently which works for both centos and RHEL, would you mind putting a conditional here please?  

I guess checking the podman version is safest since it's probably only a matter of time before centos picks up the newer podman with the required unit (until the bz mentioned by Stephen is fixed)